spacy/lang/char_classes.py

```diff
@@ -116,7 +116,7 @@
 _latin_extendedE = _latin_l_extendedE
 
 # phonetic letters - Greek, Latin, Cyrillic
-_latin_l_phonetic = r"\u0250-\u02AF\u1D00-\u1D25\u1D6B-\u1D77\u1D79-\u1D9A"
+_latin_l_phonetic = r"\u0250-\u02AF\u1D00-\u1D25\u1D6B-\u1D77\u1D79-\u1D9A\u0400-\u04FF"
```

Comment:
This isn't the right place for this character range. This is just for phonetic alphabets like IPA characters.

Reply:
Alright, I will revert this line.